cronjob.yaml

```diff
@@ -0,0 +1,26 @@
+apiVersion: batch/v1beta1
```

Comment:
@seanshahkarami can you validate this yaml?

i took a quick check against another known kubernetes cronjob that we run https://github.com/waggle-sensor/waggle-edge-stack/blob/main/kubernetes/wes-camera-provisioner.yaml

Reply:
Yes I think it should work in our k3s.

Reply:
I ran it on the nx board (k3s).